apis/userapi/userapi.yaml

```diff
@@ -1721,6 +1796,135 @@ paths:
             'INTERNAL SERVER ERROR. We track these errors automatically and try to set it right at the earliest. Try refreshing the page. If the problem persists contact us at info@sunbird.org.'
           schema:
             $ref: '#/definitions/UserErrorResponse'
+  /user/v1/get/{idType}/{id}:
+    get:
+      description: >-
+        This API is used for searching a specific user, the user search is based on the unique ID type of the user
+          - The endpoint for **Get User by User ID** is `/user/v1/get/{idType}/{id}`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+          - User can pass an optional query parameter 'fields'. The value is a comma separated list of fields whose information is required in response. e.g. fields=completeness,missingFields,topic,organisations,roles
+          - 'completeness' is to return profile completeness percentage.
+          - 'missingFields' is to return fields that are missing to achieve 100% profile completeness.
+          - 'topic' is to return list of topics that are used for announcements.
+          - 'organisations' is to return organisation and location details.
+          - 'roles' is to return list of roles with name and ID details.
+
+      summary: Get User by  ID Type
+      tags:
+        - User APIs
+      operationId: Get User by ID type
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: idType
+          type: string
+          required: true
+          description: 'idType can only be either email, phone or loginId'
+        - in: path
+          name: id
+          type: string
+          required: true
+          description: 'id can only be values of email, phone or loginId'
+        - name: ts
+          in: header
+          required: false
+          type: string
+          description: 'Timestamp at which given API request is sent.'
+        - name: X-msgid
+          in: header
+          required: false
+          type: string
+          description: 'This ID uniquely identifies a request if the same API is executed multiple times.'
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          description: 'Specify authorization key (format: Bearer api-key) received from administrator when performing given API request.'
+        - name: X-Authenticated-User-Token
```

Comment:
- name: X-Authenticated-User-Token
 not required for get user by email/phone.

Reply:
its required @manzarul

Reply:
ok